app/src/main/res/layout/dialog_contentsettings.xml

```diff
@@ -0,0 +1,32 @@
+<?xml version="1.0" encoding="utf-8"?>
+<RelativeLayout xmlns:android="http://schemas.android.com/apk/res/android"
```

Comment:
Give this resource a better name. It's not a dialog, its a fragment, and it should somehow have the "tab" in its name. "contentSettings" is to generic.